corehq/apps/registration/templates/registration/email/mobile_signup_reminder.html

```diff
@@ -0,0 +1,104 @@
+            /* Style the footer with the same color as the footer text */
+            .footer a {
+                color: #685c53;
+                -webkit-text-size-adjust: none;
+                text-decoration: underline;
+                font-weight: normal
+            }
+            /* Hide preview text on rendering */
+            #preview_text {
+                display: none;
+            }
+            /* Only supported in outlook 2000/2003 hotmail, apple mail, yahoo! mail */
+            .btn:hover {
+                background-color: #002c5f;
+                -webkit-box-shadow: 0px 0px 10px #004ebc;
+                -moz-box-shadow: 0px 0px 10px #004ebc;
+                box-shadow: 0px 0px 10px #004ebc;
+            }
+        /*]]>*/
+        </style>
+        <!-- http://www.emailon@cid.com/blog/details/C13/ensure_that_your_entire_email_is_rendered_by_default_in_the_iphone_ipad -->
+        <!--                                                                                                                     -->
+        <!--                                                                                                                     -->
+        <!--                                _|_|_|    _|                                      _|                                 -->
+        <!--                                _|    _|      _|_|_|  _|_|      _|_|_|    _|_|_|                                     -->
+        <!--                                _|    _|  _|  _|    _|    _|  _|    _|  _|    _|  _|                                 -->
+        <!--                                _|    _|  _|  _|    _|    _|  _|    _|  _|    _|  _|                                 -->
+        <!--                                _|_|_|    _|  _|    _|    _|    _|_|_|    _|_|_|  _|                                 -->
+        <!--                                                                              _|                                     -->
+        <!--                                                                            _|_|                                     -->
+        <!--                                                                                                                     -->
+        <!--                                                 Extra White Space!                                                  -->
+        <!--                                                                                                                     -->
+        <!-- - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -->
+    </head>
+    <body bgcolor="#f2f2f2" marginheight="0" marginwidth="0" topmargin="0" style="background-color: #f2f2f1; margin: 0; padding: 0; font-family: 'Helvetica Neue', 'Helvetica', 'Lucida Sans', 'Trebuchet MS', 'Arial', sans-serif;">
+
+    <!-- Preview text (text which appears right after subject) -->
```

Comment:
nit: you can use `{# this is my comment #}` for django template comments if you don't want them to show up in the rendered html.  Doesn't really matter in most cases, though.